viahtml/wsgi.py

```diff
@@ -0,0 +1,11 @@
+"""The application providing a WSGI entry-point."""
+
+from gevent.monkey import patch_all
```

Comment:
This creates all sorts of horror if it's not called before certain things, which made importing the previous `Application` a dicey business. This doesn't remove the horror, but it does quarantine it